### ISSUE TITLE
JDL import gives weird answers to an error

### ISSUE THREAD

**Issue by Tonterias** [jhipster/generator-jhipster#7996]
##### **Overview of the issue**
I do not think notificationReazon is a reserved word. I was trying to spot another problem so I paste only part of the JDL file. The real reason is that the notificationReason Enum was not included.

D:\BasuraTemporal\Borrar\BOOTSRAP\JHispter\JHipsterPress04>yo jhipster:import-jdl ./src/main/scripts/redux.jh
The JDL is being parsed.
Error: The name 'notificationReason' is a reserved keyword and can not be used as an entity field name.
Error jhipster:import-jdl ./src/main/scripts/redux.jh

ERROR! Error while parsing applications and entities from the JDL Error: The name 'notificationReason' is a reserved keyword and can not be used as an entity field name.

D:\BasuraTemporal\Borrar\BOOTSRAP\JHispter\JHipsterPress04>yo jhipster:import-jdl ./src/main/scripts/redux.jh
The JDL is being parsed.
Error: The name 'notificationEvent' is a reserved keyword and can not be used as an entity field name.
Error jhipster:import-jdl ./src/main/scripts/redux.jh

ERROR! Error while parsing applications and entities from the JDL Error: The name 'notificationEvent' is a reserved keyword and can not be used as an entity field name.

D:\BasuraTemporal\Borrar\BOOTSRAP\JHispter\JHipsterPress04>yo jhipster:import-jdl ./src/main/scripts/redux.jh
The JDL is being parsed.
Error: The name 'notificationReazon' is a reserved keyword and can not be used as an entity field name.
Error jhipster:import-jdl ./src/main/scripts/redux.jh

ERROR! Error while parsing applications and entities from the JDL Error: The name 'notificationReazon' is a reserved keyword and can not be used as an entity field name.

D:\BasuraTemporal\Borrar\BOOTSRAP\JHispter\JHipsterPress04>

##### **Motivation for or Use Case**

I do not think there is anything worng with the JDL file.

##### **Reproduce the error**

entity Blog {
    creationDate Instant required
    title String minlength(2) maxlength(100) required
    image ImageBlob
}
  
entity Post {
    creationDate Instant required
    publicationDate Instant
    headline String minlength(2) maxlength(100) required
    leadtext String minlength(2) maxlength(1000)
    bodytext String minlength(2) maxlength(65000) required
    quote String minlength(2) maxlength(1000)
    conclusion String minlength(2) maxlength(2000)
    image ImageBlob
}

entity Topic {
	topicName String minlength(2) maxlength(40) required
}

entity Tag {
	tagName String minlength(2) maxlength(40) required
}

entity Comment {
    creationDate Instant required
    commentText String minlength(2) maxlength(65000) required
    isOffensive Boolean
}

// TODO: Websockets
entity Message {
    creationDate Instant required
    messageText String minlength(2) maxlength(8000) required
    isDeliverd Boolean
}

entity Notification {
    creationDate Instant required
    notificationDate Instant
    notificationReason NotificationReason required
    notificationText String minlength(2) maxlength(100)
    isDeliverd Boolean
}

entity Profile {
    creationDate Instant required
    image ImageBlob
    gender Gender required
    phone String maxlength(20) required
    bio String maxlength(7500)
    birthdate Instant
    civilStatus CivilStatus
    lookingFor Gender
    purpose Purpose
    physical Physical
    religion Religion
    ethnicGroup EthnicGroup
    studies Studies
    sibblings Boolean
    eyes Eyes 
    smoker Smoker
    children Children
    futureChildren FutureChildren
    pet Boolean
}

entity Community {
    creationDate Instant required
    communityname String minlength(2) maxlength(100) required
    communitydescription String minlength(2) maxlength(7500) required
    image ImageBlob
    isActive Boolean
}

entity Follow {
    creationDate Instant
}

entity Blockeduser {
    creationDate Instant
}

entity Album {
    creationDate Instant required
    title String minlength(2) maxlength(100) required
}

entity Photo {
    creationDate Instant required
	image ImageBlob
}

entity Interest {
    interestName String minlength(2) maxlength(40) required
}

entity Activity {
    activityName String minlength(2) maxlength(40) required
}

entity Celeb {
    celebName String minlength(2) maxlength(40) required
}

entity Urllink {
    linkText String required
    linkURL String required
}

entity Frontpageconfig {
    creationDate Instant required
    topNews1 Long
    topNews2 Long
    topNews3 Long
    topNews4 Long 
    topNews5 Long
    latestNews1 Long
    latestNews2 Long
    latestNews3 Long
    latestNews4 Long
    latestNews5 Long
    breakingNews1 Long
    recentPosts1 Long
    recentPosts2 Long
    recentPosts3 Long
    recentPosts4 Long
    featuredArticles1 Long
    featuredArticles2 Long
    featuredArticles3 Long
    featuredArticles4 Long
    featuredArticles5 Long
    featuredArticles6 Long
    featuredArticles7 Long
    featuredArticles8 Long
    featuredArticles9 Long
    featuredArticles10 Long
    popularNews1 Long
    popularNews2 Long
    popularNews3 Long
    popularNews4 Long
    popularNews5 Long
    popularNews6 Long
    popularNews7 Long
    popularNews8 Long
    weeklyNews1 Long
    weeklyNews2 Long
    weeklyNews3 Long
    weeklyNews4 Long
    newsFeeds1 Long
    newsFeeds2 Long
    newsFeeds3 Long
    newsFeeds4 Long
    newsFeeds5 Long
    newsFeeds6 Long
    usefulLinks1 Long
    usefulLinks2 Long
    usefulLinks3 Long
    usefulLinks4 Long
    usefulLinks5 Long
    usefulLinks6 Long
    recentVideos1 Long
    recentVideos2 Long
    recentVideos3 Long
    recentVideos4 Long
    recentVideos5 Long
    recentVideos6 Long
}

##### **Suggest a Fix**

Change the error message 

##### **JHipster Version(s)**

5.0.1 because the upgrade does not let me upgrade but gives no error. That is a different story, right?

##### **JHipster configuration**

Not applicable

##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

Not applicable

##### **Browsers and Operating System**

Any

- [X] Checking this box is mandatory (this is just to show you read everything)

**Comment by MathieuAA** [comment]
Hi, I'm gonna check that.

**Comment by MathieuAA** [comment]
Is it the same thing? If so, it's already available in the master branch of the generator. Thanks Jon

**Comment by ruddell** [comment]
Yep, same issue as https://github.com/jhipster/generator-jhipster/issues/7924 and https://github.com/jhipster/generator-jhipster/issues/7861
